src/api/app/models/cloud/backend/upload_job.rb

```diff
@@ -32,8 +32,17 @@ def self.create(params)
         new(exception: exception.message)
       end
 
-      def self.all(user)
+      def self.find(job_id, user, options = {})
+        xml = ::Backend::Api::Cloud.upload_jobs([job_id])
```

Comment:
Hm, maybe ``upload_jobs`` should take care if you put in an array or single id?
(I think it should already work out of the box, no?)

Reply:
I think we should use the interface properly instead of adding code to ` upload_jobs ` to make it accept all kinds of input.
And I think it's fine that the upload_jobs expects an array of ids. So I would keep it as is.